src/main/java/io/reactivex/rxjava3/internal/operators/observable/ObservableThrottleFirstTimed.java

```diff
@@ -83,6 +97,13 @@ public void onNext(T t) {
                     d.dispose();
                 }
                 DisposableHelper.replace(this, worker.schedule(this, timeout, unit));
+            } else if (onDropped != null) {
+                try {
+                    onDropped.accept(t);
+                } catch (Throwable ex) {
+                    Exceptions.throwIfFatal(ex);
+                    upstream.dispose();
```

Comment:
This will hang the sequence because the error is swallowed. Also there is no cleanup of the worker.

Reply:
So we have to do downstream.onError(t) to propagate this, right? And .dispose() of the worker too?

Reply:
Yes.

Reply:
Should be addressed now, thx for the review